File: cpp/lv0/find_center_value.cpp
// 중앙값 구하기

#include <string>
#include <vector>
#include <algorithm>

using namespace std;

/* Answer 1
int solution(vector<int> array) {
    int answer = 0;
    
    sort(array.begin(), array.end());
    
    int medium = array.size() / 2;
    answer = array[medium];
    
    return answer;
} */

/* Answer 2
int solution(vector<int> array) {

    sort(array.begin(), array.end());

    return array[array.size() / 2];
} */

File: python/lv0/find_center_value.py
# 중앙값 구하기

'''
Answer 1

def solution(array):
    answer = 0
    
    array.sort()
    medium = int(len(array) / 2)    
    answer = array[medium]
    
    return answer

'''

'''
Answer 2

def solution(array):
    answer = 0

    a = sorted(array)
    medium = int(len(array) / 2)
    answer = a[medium]
    
    return answer

'''

'''
Answer 3

def solution(array):
    
    return sorted(array)[len(array) // 2]

'''
